src/components/icon/icon.md

```diff
@@ -1,44 +1,64 @@
-The `<Icon />` component's API is mostly self-evident and you can refer to the props table at the bottom of this page.
-However, let's look at the color properties in more detail.
-We use three different properties to apply different colors to an SVG:
+The `<Icon>` component represents our latest patterns with using SVGs.
+It is also designed to be accessible -- please read our accessibility section below!
 
-- `fill` sets the fill color
-- `stroke` sets the stroke color
-- `currentColor` is an SVG trick to provide an additional `fill` and/or `stroke` color
+Forward facing note:
 
-Basic Usage
+- `currentColor` API is being deprecated in favor of self-contained SVGs -- please use `skip` for now
+- `fill` API is being deprecated in favor of self-contained SVGs -- please use `skip` for now
+- `skip` API is being deprecated in favor of self-contained SVGs -- please use `skip` for now
+- *Self-contained are SVGs with all coloring styling applied in the SVG file
+
+### Basic Usage
 
 ```jsx
 const icon = require('../../svgs/icon-caution-fill.svg').default;
-<Icon name={icon.id} size="large" currentColor="caution" title="alert icon" />
+<Icon name={icon.id} size="large" currentColor="skip" fill="skip" stroke="skip" title="alert icon" />
 ```
-Sizes
+
+### Sizes
+
+For the most part, the `size` API is the main configurable property.
```

Comment:
We should probably mention what those sizes are (in pixel units), and how the container size is not the icon size. Encourage the double checking of design mocks to ensure that the correct spacing is used, that kind of thing.

Reply:
Good point. Lemme think about that phrasing.